features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Documents/Create.jsx

```diff
@@ -144,6 +148,7 @@ function Create(props) {
 Create.propTypes = {
     classes: PropTypes.shape({}).isRequired,
     intl: PropTypes.shape({}).isRequired,
```

Comment:
can we change intl type to func? cause i think it should be func not the shape of object

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/6759/commits/0c218762d38e7db55b9b2b6306a4a1027e6d4cb0